Description: setup a bigger window, as now we have bigger desktop resolution
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tinymain.asm
+++ b/tinymain.asm
@@ -29,9 +29,9 @@
         xor     eax,eax
         mov     al,32
         push    eax
-        mov     ax,480
+        mov     ax,600
         push    eax
-        mov     ax,640
+        mov     ax,800
         push    eax
         call    SDL_SetVideoMode
         
